docs/specviz/plugins.rst

```diff
@@ -164,6 +164,28 @@ Similarly, all of the line lists may be shown or hidden via
 :guilabel:`Plot All` and :guilabel:`Erase All`, located at the
 bottom of the plugin.
 
+.. _custom-line-lists:
+
+Importing Custom Line Lists
+---------------------------
+
+Jdaviz comes with curated line lists built by the scientific community.
+If you cannot find the lines you need, you can add your own by constructing
+an :ref:`astropy table <astropy:construct_table>`; For example::
+
+    from astropy.table import QTable
+    from astropy import units as u
+
+    my_line_list = QTable()
+    my_line_list['linename'] = ['Hbeta','Halpha']
+    my_line_list['rest'] = [4851.3, 6563]*u.AA
+    my_line_list['redshift'] = u.Quantity(0.046) # Optional
+
+    viz.load_line_list(my_line_list)
+    # Show all imported line lists
+    viz.spectral_lines
+
+
```

Comment:
I have slightly mixed feelings about this moving into the plugin since there is no UI-way to import them, but agree that at least a mention from here will help make sure that people can find the information.

Reply:
Yeah, definitely understand the mixed feelings. I think we need to have this information in one of the two sections and link to it from the other. I'm indifferent as to which section has it though.